src/Microsoft.AspNetCore.Mvc.Core/ResponseCacheAttribute.cs

```diff
@@ -78,6 +78,11 @@ public bool NoStore
         public string VaryByHeader { get; set; }
 
         /// <summary>
+        /// Gets or sets the query keys used by the response cache middleware for creating secondary vary keys.
```

Comment:
> Gets or sets the query values to vary by.

The use of response caching middleware is a detail of the filter, not the attribute.